Added feedback button functionality - opens Email client and has pres…

christiansiv opened the pull request:
…et to address (test@codersforcauses) and subject.

christiansiv committed:
Added feedback button functionality - opens Email client and has preset to address (test@codersforcauses) and subject.

alexarnold94 commented on app/src/main/java/accesscollective/uwastudentguild/com/accesscollective/MainActivity.java:
```diff
@@ -21,6 +23,9 @@
 public class MainActivity extends AppCompatActivity
         implements NavigationView.OnNavigationItemSelectedListener, OnMapReadyCallback {
 
+    String subject= "Access Collective App feedback";
```
I'd probably shift these global variables to be within the onNavigationItemSelected function - it's good practice to [avoid using global variables](http://wiki.c2.com/?GlobalVariablesAreBad)

alexarnold94 reviewed (commented):
Works and it works well, just a couple small best practice things but in terms of functionality it's spot on

christiansiv committed:
Added feedback button functionality - opens Email client and has preset to address (test@codersforcauses) and subject.

Johnn27 committed:
Added change campus functionality

Commit:
Fixed unneeded back navigation

By clearing the back stack before loading the MapFragment, it makes it more like the main page of the app (we may have to change some of the navigation later to make the MapFragment the actual main page, but we'll see)

alexarnold94 reviewed (approved):
Easy guys we're good to merge
